services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/logging/Persistence.kt

```diff
@@ -10,11 +10,13 @@ import javax.persistence.GenerationType
 import javax.persistence.Id
 import javax.persistence.Table
 
-enum class Action {
-    CREATE,
-    READ,
-    UPDATE,
-    DELETE
+enum class Action(val isChange: Boolean) {
+    CREATE(true),
+    READ(false),
+    UPDATE(true),
+    DELETE(true),
+    LOCK(false),
+    UNLOCK(false)
 }
```

Comment:
Needs explaining comments for at least boolean

Reply:
Added comments for all of that.